src/main/java/roomescape/reservation/response/ReservationResponse.java

```diff
@@ -0,0 +1,37 @@
+package roomescape.reservation.response;
+
+import java.time.format.DateTimeFormatter;
+import roomescape.reservation.model.Reservation;
+
+public class ReservationResponse {
+
+    private final Long id;
+    private final String name;
+    private final String date;
+    private final String time;
+
+    public ReservationResponse(Reservation reservation) {
+        this.id = reservation.getId();
+        this.name = reservation.getName();
+        this.date = reservation.getDate().format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+        this.time = reservation.getTime().format(DateTimeFormatter.ofPattern("HH:mm"));
```

Comment:
`DateTimeFormatter` 객체를 한 번만 만들어 두고 재사용할 수 있을 것 같네요!
```suggestion
        this.date = reservation.getDate().format(DATE_FORMATTER);
        this.time = reservation.getTime().format(TIME_FORMATTER);
```